src/libultra/os/initialize.c

```diff
@@ -1,63 +1,62 @@
 #include "global.h"
 
 typedef struct {
-    u32 ins_00; // lui     k0, 0x8000
-    u32 ins_04; // addiu   k0, k0, 0x39E0
-    u32 ins_08; // jr      k0 ; __osException
-    u32 ins_0C; // nop
-} struct_exceptionPreamble;
+    u32 inst1; // lui     $k0, %hi(__osException)
+    u32 inst2; // addiu   $k0, $k0, %lo(__osException)
+    u32 inst3; // jr      $k0
```

Comment:
fwiw the .s in the repo don't use $-regs syntax

Reply:
exceptasm.s in the oot repo does use $ so until that changes this should mirror it